### DISCUSSION TITLE
Add precision on the positioning of the common guides

### DISCUSSION THREAD

**Discussion by calogerogigante** [opentoonz/opentoonz#6505] [Ideas]
Is it possible to display a pop-up indicating the pixel during the positioning of the guide in the ruler ? Can you do that for the on-coming release, please ? It's sad to have working snapping on guides, and not have the possibility to place precisely these guides...

**Comment by RodneyBaker** [comment]
@calogerogigante Recommend closing this report as this area (the nightlies repo) is not the correct location for reporting Opentoonz issues.

**Comment by shun-iwasawa** [comment]
I'll transfer this to OpenToonz repo.
